infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -376,13 +375,15 @@ private void verifyLogs() {
 
     @Override
     public void sync(boolean force) throws IOException {
-        if (force) {
-            for (FileChannel ch : channelsToSync) {
-                ch.force(true);
+        for (SegmentHandle sh : segmentsToSync) {
```

Comment:
if `force` is false no need to go through the collection. So prev version is better